modules/installation-alibaba-limits.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/installing-alibaba-account.adoc
+
+[id="installation-alibaba-limits_{context}"]
+= Alibaba account limits
+
+.Context
+We have historically included a section that details the cloud provider's default component limits that might affect a cluster installation and why.[1] [2] Even if we do not believe default limits will affect a cluster installation, we have provided this type of information as a reference point [3].
```

Comment:
@mtulio @jianli-wei @kwoodson @joelddiaz Can you help with this? Are there specific quotas, limitations, etc that we want to include?

Reply:
Removed this module from the assembly. Saving the module in case we want to put it back.